tests/test-data.md

```diff
@@ -0,0 +1,67 @@
+# nebula
+
+## prepare data
```

Comment:
👍🏻, it's really nice to have data preparation part, together with the test code in this way